corehq/apps/analytics/tasks.py

```diff
@@ -87,6 +91,7 @@ def _hubspot_post(url, data):
             data=data,
             headers=headers
         )
+        logger.debug('Sent this data to HS: %s \nreceived: %s'.format(data, req.json))
```

Comment:
do you not want to convert data to json first? and maybe use `indent=2` or something?